__tests__/tweet-routes.test.js

```diff
@@ -6,22 +6,31 @@ const connect = require('../lib/utils/connect');
 const mongoose = require('mongoose');
 const Tweet = require('../lib/models/tweet.js');
 
-
 describe('app routes', () => {
-    beforeAll(() => {
-      connect();
-    });
-  
-    beforeEach(() => {
-      return mongoose.connection.dropDatabase();
-    });
-  
-    afterAll(() => {
-      return mongoose.connection.close();
-    });
+  beforeAll(() => {
+    connect();
+  });
+  beforeEach(()=> {
+    return mongoose.connection.dropDatabase();
+  });
+  afterAll(()=> {
+    return mongoose.connection.close();
+  });
+
+  //test post route 
+  it('creates a comment', async() => {
+    const tweet = await Tweet.create({ handle: 'Black Phillip', text: 'Wouldst thou like to live deliciously?' });
 
-    it('gets a tweet by id', () => {
-        return Tweet.create({
-            
-        })
-    }
\ No newline at end of file
+    return request(app)
+      .post('/api/v1/comments/')
+      .send({ tweetId: tweet._id, handle: 'Pinhead', text: 'We have such sights to show you!' });
+  //     .then(res => {
```

Comment:
You can comment this back in. In your app.js you'll need to add your tweets route so, add something like this — `app.use('/api/v1/tweets', require('./routes/tweet-routes'));`